### PR TITLE
Add Ombi support

### PR DESCRIPTION
**Pull request by mgiljum**
Ombi allows for users to request movies, TV show, albums, etc.

Their releases only come in zip format, so the download script uses wget/unzip using a temporary folder rather than piping curl to tar. It may be able to cleaned up some.

### CODE REVIEW

File: ombi/ombi.init [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+### END INIT INFO
+ 
+ 
+# Documentation available at
+# http://refspecs.linuxfoundation.org/LSB_3.1.0/LSB-Core-generic/LSB-Core-generic/iniscrptfunc.html
+# Debian provides some extra functions though
+. /lib/lsb/init-functions
+ 
+DAEMON_NAME="Ombi"
+DAEMON_USER=htpcguides
```
Comment (TommyE123) [inline ombi/ombi.init]: Should be DAEMON_USER=ReplaceMe

File: ombi/ombi-init-update.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+# Script Name: AtoMiC Ombi init.d update
+# Author: TommyE123
+# Publisher: http://www.htpcbeginner.com
+#
+
+# DO NOT EDIT ANYTHING UNLESS YOU KNOW WHAT YOU ARE DOING.
+echo 'Updating init.d file'
+
+sudo sed -i "s@RUNASUSER=root@RUNASUSER=$UNAME@g" /etc/init.d/$APPNAME || { echo -e $RED'Replacing RUN_AS failed.'$ENDCOLOR ; exit 1; }
+sudo sed -i "s@RUNASGROUP=root@RUNASGROUP=$UNAME@g" /etc/init.d/$APPNAME || { echo -e $RED'Replacing RUNASGROUP failed.'$ENDCOLOR ; exit 1; }
```
Comment (TommyE123) [inline ombi/ombi-init-update.sh]: Remove this line as not required.

File: ombi/ombi-init-update.sh [hunk 3]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Script Name: AtoMiC Ombi init.d update
+# Author: TommyE123
+# Publisher: http://www.htpcbeginner.com
+#
+
+# DO NOT EDIT ANYTHING UNLESS YOU KNOW WHAT YOU ARE DOING.
+echo 'Updating init.d file'
+
+sudo sed -i "s@RUNASUSER=root@RUNASUSER=$UNAME@g" /etc/init.d/$APPNAME || { echo -e $RED'Replacing RUN_AS failed.'$ENDCOLOR ; exit 1; }
```
Comment (TommyE123) [inline ombi/ombi-init-update.sh]: should be DAEMON_USER=ReplaceMe

File: ombi/ombi-uninstaller.sh [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Script Name: AtoMiC Radarr Uninstaller
```
Comment (TommyE123) [inline ombi/ombi-uninstaller.sh]: Change to: Script Name: AtoMiC Ombi Uninstaller

**Comment by TommyE123** [comment]
Hi Mathew,

I Couldnt tell you how pleased I was when saw someone had added a pull request this morning let alone a new application! I've been helping out for a few months now and this is the first time anyone has added anything other then me and Anand (he runs the site)!

I've looked at what you've added and am 99% happy with everything you've done. Honestly great effort and really really appreciated. However there is a couple of tiny things that need fixing if you have the time to quickly adjust them which i've hightlighted.

Just out of interest does the access details work? 
PORTSEARCH='<port>' appears to be the same as the Radarr setting i'm guessing you used as a template to get this working. It also reads from whatever you have the APPSETTINGS set to to find it.

Also did you want to add your name instead of mine to the script author?

Thanks again!!
Tom

**Comment by mgiljum** [comment]
Glad I could help! Love the script and use it all the time to run updates, etc. I did in fact use Radarr as a template (they both use Mono). The port search function works fine, but grabbing the username doesn't - both Radarr Sonarr and this new app use sqlite for their settings. I could write a query for them eventually but if you're okay with leaving it as is for now I think I'll just leave it.

I'll get those minor changes done shortly.

**Comment by TommyE123** [comment]
Yep, Just leave it for now that's fine. If you want to write something to query that file for those apps it would be fantastic. I'm currently putting the finishing touches to Subsonic and Madsonic. I also keep finding little niggles I want to fix\improve all over the place!

Thanks again for your time to add this.

Tom

**Comment by mgiljum** [comment]
Those changes should be good to go, but check that line in `ombi/ombi-init-update.sh`. I believe it's  correct.

File: ombi/ombi-init-update.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Script Name: AtoMiC Ombi init.d update
+# Author: mgiljum
+# Publisher: http://www.htpcbeginner.com
+#
+
+# DO NOT EDIT ANYTHING UNLESS YOU KNOW WHAT YOU ARE DOING.
+echo 'Updating init.d file'
+
+sudo sed -i "s@DAEMON_USER=ReplaceMe@RUNASUSER=$UNAME@g" /etc/init.d/$APPNAME || { echo -e $RED'Replacing RUN_AS failed.'$ENDCOLOR ; exit 1; }
```
Comment (TommyE123) [inline ombi/ombi-init-update.sh]: DAEMON_USER=\"ReplaceMe\"@DAEMON_USER=\"$UNAME\"@

it also needs a \ before " on the 4 " (see the deluged default update as an example) i can't seem to get it to show up on the comment!

File: ombi/ombi.init [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+### END INIT INFO
+ 
+ 
+# Documentation available at
+# http://refspecs.linuxfoundation.org/LSB_3.1.0/LSB-Core-generic/LSB-Core-generic/iniscrptfunc.html
+# Debian provides some extra functions though
+. /lib/lsb/init-functions
+ 
+DAEMON_NAME="Ombi"
+DAEMON_USER=ReplaceMe
```
Comment (TommyE123) [inline ombi/ombi.init]: DAEMON_USER="ReplaceMe"

File: ombi/ombi-init-update.sh (hunk 5 above)
Comment (mgiljum) [inline ombi/ombi-init-update.sh]: Got it, thanks

**Comment by TommyE123** [comment]
Perfect! As you can see i've merged it up.

If you did want to continue to help and wanted to add anything else just give me a shout. I've started adding feature requests to the main issue log with what i'm looking at. So if you did want to add anything  then you could just do that as well so i don't end up potentially duplicating your work!!!

Thanks again. 
Glad you find the tool useful as well.

Tom

**Comment by mgiljum** [comment]
Glad I could help. I added an issue for the db query on apps that use sql or sqlite, and I've made a branch on my fork to experiment with that a bit.

**Comment by mgiljum** [comment]
If you want, you could update the project description to include Ombi.

**Comment by TommyE123** [comment]
There appears to be an issue with detecting the systemd file in the access details "grep: : No such file or directory" not checked whats causing it but complete guess is missing constant not set?

**Pull request merged**